<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogfileConverter</class>
 <widget class="QMainWindow" name="LogfileConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logfile converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Convert base64 encoded logfiles created by the BMU into CSV files.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Input file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputPath"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnInputFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Output file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputPath"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnOutputFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progress">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConvert">
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="status">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
